spec/user_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../lib/person.rb'
+require_relative '../lib/user.rb'
+
+describe User do
+
+  context '#print_age_based_message' do
```

Comment:
These should be `describe` blocks -- you're describing how a method should work. Think of `context` as a scenario.